docs/source/notebooks/Image_Tut.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classification with Cleanlab, PyTorch & Skorch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will use Cleanlab to find potential label errors in the MNIST handwritten digit image dataset. This dataset contains 60,000 images of handwritten digits from 0 to 9. We'll find *more than a thousand* highly suspicious examples with label errors and edge cases by the end of this tutorial. \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:**\n",
+    "\n",
+    "- Build a simple PyTorch neural net and wrap it with Skorch to make it scikit-learn compatible.\n",
+    "\n",
+    "- Compute the cross-validated predicted probabilities, ``pyx``, sometimes known as the out-of-sample predicted probabilities.\n",
+    "\n",
+    "- Generate a list of potential label errors with Cleanlab's ``get_noise_indices``.\n",
+    "\n",
+    "- Build and train a more robust model than the vanilla model after adapting it with Cleanlab's ``LearningWithNoisyLabels``. \n",
+    "\n",
+    "**Data:** https://www.openml.org/d/554"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "``%%capture`` is a magic function to hides the cell's output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "\n",
+    "%pip install cleanlab pandas matplotlib torch torchvision skorch"
```

Comment:
What are pros/cons of installing the dependencies in notebook cell vs. just explaining in text that user should install them (can give them the command as well)?

Reply:
For instance, the user viewing `master` version of these docs should not actually be `pip installing cleanlab` at all, so that's incorrect.

Reply:
We also later want these notebooks to be automatically executed using whatever version of cleanlab a current PR is using, so again `pip installing cleanlab` in the cell will not work there.

Reply:
The intention of using `pip install cleanlab` in the code cell is to create a self-contained nb where a user can simply click "Run All" in Jupyter/Colab and see the output. But i see ur concerns when catering to these 3 scenarios:

**Scenario 1**: A user wanting to use the `stable` release of `cleanlab`
**Scenario 2**: A user wanting to use a specific branch/tag of `cleanlab`, e.g. `master`
**Scenario 3**: A contributor edited the `cleanlab` codebase, raising a PR

In the tutorials, instead of just `pip install cleanlab`, how about we change to:
```
import os
import sys

if os.path.exists("./cleanlab"):
    sys.path.append('.')
else:
    pip install cleanlab
```
So if its **Scenario 3**, it would be building in the CI server or locally with the `./cleanlab` dir, hence it will just use the edited `cleanlab` codebase, else, it would cater for **Scenario 1**. 

To address **Scenario 2**, we can add a note in the tutorial saying:

> **Looking for a specific branch or tag of cleanlab?**
> Run `pip install git+https://github.com/cleanlab/cleanlab.git@TAG-OR-BRANCH-NAME`

Reply:
How about we just have `pip install cleanlab` (and all the other packages) in a non-executable "cell" of the notebook?  On the CI server we should then just install these packages as dependences in the docs requirements.  

What I mean by this idea is to just have a markdown cell but make it look like code, eg:

If you don't have these packages installed yet, you'll first need to run the following:
```
pip install cleanlab ...
```

Reply:
done